apps/bluetooth/app.js

```diff
@@ -446,7 +446,9 @@ module.exports = function (activity) {
         }
         break
       case protocol.CONNECTION_STATE.DISCONNECTED:
-        if (currentSkillName !== 'bluetooth') {
+        if (currentSkillName === 'bluetooth_call') {
+          onCallStateChangedListener(protocol.CALL_STATE.IDLE)
+        } else if (currentSkillName === 'bluetooth_music') {
```

Comment:
Should there be a fallback for skills not matched `bluetooth_call` or `bluetooth_music`?

Reply:
I think not need. There are 3 skills, if match with `bluetooth`, nothing to be done.